Makefile

```diff
@@ -119,6 +123,7 @@ license-check: bin/licensei ## Run license check
 	bin/licensei header
 	cd backend && ../bin/licensei check --config=../.licensei.toml
 	cd plugins/gateway/kong && ../../../bin/licensei check --config=../../../.licensei.toml
+	cd plugins/gateway/tyk && ../../../bin/licensei check --config=../../../.licensei.toml
```

Comment:
you need to run in inside the v3.2.2 folder
`cd plugins/gateway/tyk/v3.2.2 ../../../../bin/licensei check --config=../../../../.licensei.toml`